documentation/02_handbook/21-debugger-interaction.html.md

```diff
@@ -0,0 +1,39 @@
+```
+title: "Debugger - Interaction"
+```
+
+The interaction tool (enabled by the ![](../images/02_handbook/debugger/icons/interactive.png) button in the debugger overlay) allows you to change game elements, e.g. move sprites, while the game is still running.
+
+It is highly recommended, however, that you pause the game using VCR's pause button ![](../images/02_handbook/debugger/icons/pause.png) before using the interaction tool. You can always resume the game by clicking VCR's play button ![](../images/02_handbook/debugger/icons/arrowRight.png) after you are done with the interaction. If the game is not paused during the interaction, objects affected by acceleration, for instance, will continue to move, making your work harder.
+
+## Interaction tools
+
+When the interaction is enabled, a toolbar containing a few related tools is displayed on the left side of the screen. Below is a description of each tool.
+
+### Pointer
+
+The pointer tool ![](../images/02_handbook/debugger/icons/cursorCross.png) allows you to select game elements. In order to use it, simply click elements on the screen:
+
+![](../images/02_handbook/debugger/interaction-pointer-simple-select.gif)
+
+Currently only elements that extend `FlxSprite` can be selected (which exclude tiles of a tilemap, for instance). Elements that belong to your selection will be highlighted in red. If you click an empty space, the selection will be discarded.
```

Comment:
Hm, I guess `FlxTilemap` extends `FlxObject`, so that would have led to some unexpected behavior.. I can definitly think of valid use cases for wanting to manipulate FlxObjects, but they're usually not subclassed by users.